releasenotes/notes/adding-CID-detection-PDFMinerToDocument-0195a929d64cd502.yaml

```diff
@@ -0,0 +1,5 @@
+---
+enhancements:
```

Comment:
I agree that this is more a small feature than a bug fix. You could update the PR title to feat instead of fix to reflect that.